Modules/TreasureHunt/Presenters/TreasureHuntPresenter.php

```diff
@@ -31,14 +31,20 @@ public function createComponentIntroForm()
     {
         $form = $this->registerFormFactory->create();
         $form->onSuccess[] = function (Form $form, $values) {
+            // $values come in trimmed :)
```

Comment:
Toto nemusí být vždy pravda - záleží na konkrétních formulářových prvcích a případně jejich nastavení

Komentář nic nepřináší, odeber ho prosím